lib/grape/router/route.rb

```diff
@@ -56,6 +57,15 @@ def route_path
         pattern.path
       end
 
+      def hidden
```

Comment:
Are we really just trying to make grape-swagger happy here? The whole `hidden` thing is not a Grape feature, just happens to be an option that grape-swagger "extends". Maybe it's better to expose `options` instead and let grape-swagger deal with it?